processor/spanmetricsprocessor/processor.go

```diff
@@ -53,8 +56,34 @@ var (
 	defaultLatencyHistogramBucketsMs = []float64{
 		2, 4, 6, 8, 10, 50, 100, 200, 400, 800, 1000, 1400, 2000, 5000, 10_000, 15_000, maxDurationMs,
 	}
+
+	dimensionsCacheSize   = stats.Int64("dimensions_cache_size", "size of LRU cache of dimension key-value maps keyed by a unique identifier.", stats.UnitDimensionless)
+	uniqueTimeSeriesCount = stats.Int64("unique_time_series_count", "number of unique time series.", stats.UnitDimensionless)
+	spanIngestedCount     = stats.Int64("span_ingested_count", "number of spans ingested", stats.UnitDimensionless)
+	spanProcessedCount    = stats.Int64("span_processed_count", "number of spans processed", stats.UnitDimensionless)
```

Comment:
Prefixing these with `spanmetrics` will make them much easier to discover
```suggestion
	dimensionsCacheSize   = stats.Int64("spanmetrics_dimensions_cache_size", "size of LRU cache of dimension key-value maps keyed by a unique identifier.", stats.UnitDimensionless)
	uniqueTimeSeriesCount = stats.Int64("spanmetrics_unique_time_series_count", "number of unique time series.", stats.UnitDimensionless)
	spanIngestedCount     = stats.Int64("spanmetrics_span_ingested_count", "number of spans ingested", stats.UnitDimensionless)
	spanProcessedCount    = stats.Int64("spanmetrics_span_processed_count", "number of spans processed", stats.UnitDimensionless)
```